src/test/java/org/generationcp/commons/util/filter/DatabaseConnectionFilterTest.java

```diff
@@ -9,8 +9,8 @@
 import org.generationcp.middleware.manager.api.WorkbenchDataManager;
 import org.generationcp.middleware.pojos.workbench.Project;
 import org.generationcp.middleware.support.servlet.MiddlewareServletContextListener;
+import org.hibernate.Session;
```

Comment:
Is `DatabaseConnectionFilter` class still relevant?

Reply:
@darla-leafnode  it is used in GDMS and we did not dig into that project that much